File: leetcode/3005. 最大频率元素计数.cc
class Solution {
public:
    int maxFrequencyElements(vector<int>& nums) {
        map<int, int> m;
        for (auto &i : nums) {
            m[i]++;
        }
        int res = 0;
        for (auto &i : m) {
            res = max(res, i.second);
        }
        int k = 0;
        for (auto &i : m) {
            if (i.second == res) {
                k += i.second;
            }
        }
        return k;
    }
};

File: leetcode/3005. 最大频率元素计数.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer, Integer> m = new HashMap<>();
        for (int n : nums) {
            m.put(n, m.getOrDefault(n, 0) + 1);
        }
        int res = 0;
        for (int v : m.values()) {
            res = Math.max(res, v);
        }
        int k = 0;
        for (int v : m.values()) {
            if (v == res) {
                k += res;
            }
        }
        return k;
    }
}